docker-entrypoint.sh

```diff
@@ -44,6 +44,13 @@ if [[ "$1" == "kong" ]]; then
   if [[ "$2" == "docker-start" ]]; then
     kong prepare -p "$PREFIX" "$@"
 
+    # remove all dangling sockets in $PREFIX dir before starting Kong
+    for localfile in "$PREFIX"/*; do
```

Comment:
I doubt it would ever come up (certainly not with any sockets that Kong currently creates), but worth noting that this will miss files that start with `.` (i.e. `$PREFIX/.my_socket.sock`) depending on the value of `dotglob` ([docs](https://www.gnu.org/software/bash/manual/html_node/The-Shopt-Builtin.html) since they are supposed to be "hidden" by default.

...bash is so finicky and weird.